.github/workflows/check-yaml-files-generation.yml

```diff
@@ -0,0 +1,28 @@
+name: "Check generated YAML files"
+on: pull_request
+jobs:
+  changelog:
+    runs-on: ubuntu-latest
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.8
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.8'
+    - name: Install dependencies
+      run: |
+        pip install -Ur requirements.txt -Ur requirements_dev.txt
+        python setup.py develop
```

Comment:
Nitpick: Nowadays, I think it's more common to use `pip install -e .` rather than `python setup.py develop`. I'm not sure there's any real difference in behavior, though.

Reply:
Can this me merged with the line above? Or does it still need to be two commands?

Reply:
Couldn't get it to work so reverted back to `python setup.py develop`.

Reply:
Interesting. No worries!